### ISSUE TITLE
Where to download the ground truth of the segmentation masks

### ISSUE THREAD

**Issue by JessLittle-art** [lvis-dataset/lvis-api#19]
## ❓ Questions and Help
I downloaded training set of 1,270,141 instances (1 GB), and 100,170 images (18 GB) from the website "https://www.lvisdataset.org/dataset". I wonder where is the ground truth of the segmentation masks of these 100,170 images? Thanks.

**Comment by zaiquanyang** [comment]
> ## ❓ Questions and Help
> I downloaded training set of 1,270,141 instances (1 GB), and 100,170 images (18 GB) from the website "https://www.lvisdataset.org/dataset". I wonder where is the ground truth of the segmentation masks of these 100,170 images? Thanks.

Have you already found annotations?

**Comment by WYYYDSYYDS** [comment]
me，too
